app/src/main/java/org/fossasia/openevent/events/NoInternetEvent.java

```diff
@@ -0,0 +1,7 @@
+package org.fossasia.openevent.events;
+
+/**
+ * Created by MananWason on 8/4/2015.
+ */
+public class NoInternetEvent {
```

Comment:
Failure event does not mean No Internet. There can be other kinds of failures. You should have correct snackbar message. Whether server is down or internet is down.